app/app.go

```diff
@@ -120,15 +130,15 @@ var (
 		stakingModule{},
 		mintModule{},
 		distr.AppModuleBasic{},
-		gov.NewAppModuleBasic(getGovProposalHandlers()...),
+		gov.NewAppModuleBasic(getGovProposalHandlers()),
 		params.AppModuleBasic{},
 		crisisModule{},
 		slashing.AppModuleBasic{},
 		feegrantmodule.AppModuleBasic{},
-		ibc.AppModuleBasic{},
+		// ibc.AppModuleBasic{},
 		upgrade.AppModuleBasic{},
 		evidence.AppModuleBasic{},
-		transfer.AppModuleBasic{},
+		// transfer.AppModuleBasic{},
```

Comment:
Same as above.

Reply:
Actually maybe we should just create an issue and somehow make sure we won't miss those places.

Reply:
:+1: good idea

https://github.com/celestiaorg/celestia-app/issues/412

https://github.com/celestiaorg/celestia-app/pull/399/commits/3a58c64fdc45e87b419df7dff13b32c5cf991fcf